wrapper-specification.md

```diff
@@ -350,17 +350,24 @@ Method name | Return type | Arguments | Description
 On    | void | None | Turns the led on by setting its brightness to the maximum level.
 Off   | void | None | Turns the led off.
 Flash | void | ON interval: Number, OFF interval: Number | Enables `timer` trigger and sets `delay_on` and `delay_off` attributes to the provided values (in milliseconds).
+Set Brightness Pct | void | Percent: Number | Sets the LED's brightness to the given percentage (0-1) of the maximum.
```

Comment:
I was considering that, but since we can only write integers to the LED file, if the max is 10 and I write 0.25, reading it again will return 0.3. Do you think that we should accept this and witch it?